File: RepoTransBench/target_projects/Python/Go/navdeep-G_showme/public_tests/test_public_docs_test.go
package public_tests

import (
	"strings"
	"testing"
)

func GetDoc() string {
	// Simulate the __doc__ string of showme/core
	return "Package showme core documentation - showme provides Python decorators for tracing/testing."
}

func TestPublicDocs(t *testing.T) {
	doc := GetDoc()
	if !strings.Contains(strings.ToLower(doc), "showme") &&
		!strings.Contains(strings.ToLower(doc), "core") {
		t.Error("Doc should contain 'showme' or 'core'")
	}
}

File: RepoTransBench/target_projects/Python/C++/navdeep-G_showme/public_tests/test_public_docs_test.cpp
#include <gtest/gtest.h>
#include "showme/core.h"
#include <string>

namespace {

TEST(PublicDocs, CoreDocstringContainsShowmeOrCore) {
    // Simulating: assert "showme" in doc.lower() or "core" in doc.lower()
    std::string doc = "showme core library provides tracing, timing, and utilities";
    auto doc_lower = doc;
    std::transform(doc_lower.begin(), doc_lower.end(), doc_lower.begin(), ::tolower);
    EXPECT_TRUE(doc_lower.find("showme") != std::string::npos || 
                doc_lower.find("core") != std::string::npos);
}

}